lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -210,17 +227,44 @@ def _get_saved_grade(self, subsection, course_structure, course):  # pylint: dis
                 )
                 subsection_grade = SubsectionGrade(subsection)
                 subsection_grade.load_from_data(model, course_structure, self._scores_client, self._submissions_scores)
+                log.warning(
+                    "Loaded saved grade for course with id {0}, version {1}, subsection {2}. grade: {3}".format(
+                        unicode(course.id),
+                        unicode(course.course_version),
+                        unicode(subsection.location),
+                        subsection_grade.grade_data_as_unicode()
+                    )
+                )
                 return subsection_grade
             except PersistentSubsectionGrade.DoesNotExist:
+                log.warning("Could not find saved grade for course with id {0}, version {1}, subsection {2}.".format(
+                    unicode(course.id),
+                    unicode(course.course_version),
+                    unicode(subsection.location),
+                ))
                 return None
 
     def _save_grade(self, subsection_grade, subsection, course):  # pylint: disable=unused-argument
         """
         Updates the saved grade for the student and subsection.
         """
+        log.info("attempting to save grade for course {0}, version {1}, subsection {2}. computed grade: {3}".format(
```

Comment:
My thought was that if there are any issues saving the grade, we will have this lower-level log statement to help track when we are trying to save the grade.